shame_script.py

```diff
@@ -277,4 +270,7 @@ async def shame_passthrough(
         )
 
 
-bot.run(DISCORD_TOKEN, log_handler=None)
+if __name__ == "__main__":
```

Comment:
i don't believe the way we run the bot right now - gunicorn - will activate `if __name__ == "__main__":`
from what I recall it doesn't run the same way as when we run it as a script

might be worth taking a deeper look at if this will work or not

Reply:
oh interesting, I'll look into it